index.md

```diff
@@ -41,6 +41,21 @@ communication. If the other party is away, and you leave before they come back,
 they can still answer your question, instead of just staring at a "Hello" and
 wondering what they missed.
 
+Some people like to probe with an initial "hello" to verify whether the presence 
+status shown is true, hoping they can determine whether the recipient is near a device,  
+then will immediately follow up with a call. Don't do that either. It's rude 
+to assume that because someone is able and willing
+to answer a chat message that they are also available for a voice or video call.
+
+Instead of sending a call probe, or asking "hey, do you have time for a call?" 
+(which is better but not great), you can try something like this: 
+<pre>
+2010-07-19 12:32:12 you: Hi -- do you have X minutes for a quick call to talk about XYZ? [etc...]
+</pre>
+
+This does three things: it time bounds the request, it gives the recipient some context, and it helps you
```

Comment:
Could you limit line length to 79 characters please?

Reply:
Done